pkg/minikube/cluster/cluster.go

```diff
@@ -48,6 +48,7 @@ import (
 
 var (
 	certs = []string{"apiserver.crt", "apiserver.key"}
+	logsCmd = "docker logs origin"
```

Comment:
@hferentschik That was the first check which I did before this PR :) yes it have same name as `origin` there.